### DISCUSSION TITLE
Data Capture Event for Digitizing Riverscapes

### DISCUSSION THREAD

**Discussion by leallysmith** [Riverscapes/QRiS#181] [Ideas & Feature Requests]
Hey Everyone,

Yesterday the ETAL lab met and discussed QRiS. @joewheaton pointed out that it could be really useful to add a data capture event for our method of digitizing riverscapes. Presently the issue we've been running into is that the process isn't very streamlined and consistent across technicians as we are all generating shapefiles independently and then applying symbologies. The problem this has caused is that occasionally one technician may use a double field and another uses an integer field which causes problems later on with combining shapefiles. This seems like the exact problem QRiS can solve for the lab. This is also definitely something we could use and would help shave a little time off digitizing. We just recently wrapped up mapping ~80 sites for a LCT project and I've heard there should be more this year.

I've [linked a google doc](https://docs.google.com/document/d/10IKopN1gDufCflItdEPjOzrOjzYp5hHpO2IO35zVeSE/edit?usp=sharing) Paige created that contains the LCT structure for mapping and then a more generic format that can be used for other projects. I'm not sure if this would have to be 2 separate data capture events or if you could incorporate them both as one. 

[Link to symbology](https://usu.box.com/s/gnnz887woonji63thadx59j792lx3dwk)

Video that shows the process:

[![Video Link](http://img.youtube.com/vi/D67CI_XYbqc/0.jpg)](http://www.youtube.com/watch?v=D67CI_XYbqc "")

**Comment by joewheaton** [comment]
@leallysmith I am sorry I missed this really interesting post and that helpful 

@philipbaileynar and @KellyMWhitehead have you looked at this? Especially the [layer list](https://docs.google.com/document/d/10IKopN1gDufCflItdEPjOzrOjzYp5hHpO2IO35zVeSE/edit) and the fact they have symbology for most of these. @KellyMWhitehead I am going to try and integrate this with that layer list you gave me in the Google Doc last week. 

@philipbaileynar you need to watch [this video](http://www.youtube.com/watch?v=D67CI_XYbqc) if you have not. @leallysmith highlights the nuances of their workflow, the pain points, and highlights ways QRiS can help. Lets you and I watch this, then maybe come up with a plan to address and then you and/or @KellyMWhitehead can work with @leallysmith to action it.

**Comment by joewheaton** [comment]
BTW... @KellyMWhitehead if you can't access that Box folder, here are the QMLs. 
[etal_Symbology-selected.zip](https://github.com/Riverscapes/QRiS/files/11210214/etal_Symbology-selected.zip)
